v2/update_instance.go

```diff
@@ -43,19 +48,33 @@ func (c *client) UpdateInstance(r *UpdateInstanceRequest) (*UpdateInstanceRespon
 	}
 	switch response.StatusCode {
 	case http.StatusOK:
-		if err := c.unmarshalResponse(response, &struct{}{}); err != nil {
+		responseBodyObj := &updateInstanceResponseBody{}
+		if err := c.unmarshalResponse(response, responseBodyObj); err != nil {
 			return nil, HTTPStatusCodeError{StatusCode: response.StatusCode, ResponseError: err}
 		}
 
-		return &UpdateInstanceResponse{}, nil
+		var opPtr *OperationKey
+		if responseBodyObj.Operation != nil {
+			opStr := *responseBodyObj.Operation
+			op := OperationKey(opStr)
+			opPtr = &op
+		}
+
+		userResponse := &UpdateInstanceResponse{
+			Async:        false,
+			DashboardURL: responseBodyObj.DashboardURL,
```

Comment:
shouldn't this be dependent on a version of the API? ie, `>= 2.14` ?

Reply:
If the api is an older version, the response won't contain it so it will be set to nil when we unmarshal the response on line 52. This this will then just be set to nil. Is that insufficient? I can change it to explicitly check.